src/api/js-api.js

```
@@ -207,7 +207,13 @@ function init() {
               if (scriptEl.src) {
                 nextPromise = fetchExternalMap(scriptEl.src);
               } else {
-                nextPromise = Promise.resolve(JSON.parse(scriptEl.textContent));
+                let parsedContent;
+                try {
+                  parsedContent = JSON.parse(scriptEl.textContent);
+                } catch (e) {
```

Is swallowing this json parse error ok? The user should be aware if their import map is invalid.